lib/internal/streams/end-of-stream.js

```diff
@@ -38,6 +38,16 @@ const {
   willEmitClose: _willEmitClose,
 } = require('internal/streams/utils');
 
+const {
+  isBrandCheck,
+  kState,
+} = require('internal/webstreams/util');
```

Comment:
Unfortunately, this will break `readable-stream` I think you need to inverse the dependency and have webstream require symbols from stream.

FYI @mcollina

Reply:
As in having the symbols defined in `internal/streams/util` and webstreams requiring from there?

Reply:
Exactly. Or at least e.g. `kClosedPromise` symbol.

Reply:
Actually, we might even need to use a `SymbolFor('nodejs.webstream.closed')`. Similar to https://github.com/nodejs/node/pull/45671.

Reply:
ok so would try to do the following, define `SymbolFor('nodejs.webstream.closed')` in `internal/streams/util` and use that instead of storing closed in the kState?

Reply:
Have implemented this could you check?